Source/WebCore/platform/MediaStrategy.h

```diff
@@ -78,11 +78,21 @@ class WEBCORE_EXPORT MediaStrategy : public CanMakeThreadSafeCheckedPtr<MediaStr
     virtual bool enableWebMMediaPlayer() const { return true; }
     virtual bool isWebMediaStrategy() const { return false; }
 
+#if ENABLE(WIRELESS_PLAYBACK_MEDIA_PLAYER)
+    void setWirelessPlaybackMediaPlayerEnabled(bool);
+    bool wirelessPlaybackMediaPlayerEnabled() const;
```

Comment:
this will enabled the ability to turn it on in WK1 ; maybe `MediaStrategy::wirelessPlaybackMediaPlayerEnabled` should be made virtual, returning false by default and the actual code put in WebKitLLWebMediaStrategy

Reply:
As we talked about offline, I don't think this is necessary. UnifiedWebPreferences is the source of truth for this feature flag, and there it will remain off by default in WebKitLegacy (also, only WebKit2 code calls `setWirelessPlaybackMediaPlayerEnabled()`).